### PR TITLE
Added jhiHasNotAuthority directive

### PR DESCRIPTION
**Pull request by alefz3r0**
- Please make sure the below checklist is followed for Pull Requests.

- [ ] [Travis tests](https://travis-ci.org/jhipster/generator-jhipster/pull_requests) are green
- [ ] Tests are added where necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: generators/client/templates/angular/src/main/webapp/app/shared/auth/has-not-authority.directive.ts.ejs [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+ *
+ *     <some-element *jhiHasNotAuthority="['ROLE_ADMIN', 'ROLE_USER']">...</some-element>
+ * ```
+ */
+@Directive({
+    selector: '[<%=jhiPrefix%>HasNotAuthority]'
+})
+export class HasNotAuthorityDirective {
+
+    private authorities: string[];
```
Comment (deepu105) [inline generators/client/templates/angular/src/main/webapp/app/shared/auth/has-not-authority.directive.ts.ejs]: Please add a test for this in the generated code

**Comment by deepu105** [comment]
@alefz3r0 this is open for sometime now, are you available to add tests for this?

**Comment by deepu105** [comment]
closing this as there is no response from @alefz3r0

**Comment by vidhya03** [comment]
this is useful, i just copied the code and tried in my production app. Please point me the tests. so the someone can adopt.

**Comment by deepu105** [comment]
@vidhya03 the tests needs to be generated for the app. Look at existing Jest tests for Angular

**Pull request closed without merging**